### PR TITLE
Setup Danger

### PR DESCRIPTION
**Pull request by rnystrom**
## Changes in this pull request

Getting some basic Danger automation up and running. Should see comments from @iglistkit-bot.

Issue fixed: #394

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,6 +46,7 @@ env:
 
 before_install:
   - gem install slather --no-rdoc --no-ri --no-document --quiet
+  - gem install --version '~> 4.3'
```
Comment (jessesquires) [inline .travis.yml]: install wat... lol ?

shouldn't this be `gem install danger` ?

File: Dangerfile [hunk 2]
```diff
@@ -0,0 +1 @@
+message('Hello, world!')
```
Comment (jessesquires) [inline Dangerfile]: seems legit.

File: .travis.yml (hunk 1 above)
Comment (rnystrom) [inline .travis.yml]: rofl, maybe

**Comment by rnystrom** [comment]
Sick.

Ok setting up some basic rules.

**Comment by rnystrom** [comment]
LOVE that it edits the original post. 👏👏👏

**Comment by jessesquires** [comment]
nice

**Comment by orta** [comment]
It's like it was never even in this comment thread

**Pull request closed without merging**